package.json

```
@@ -28,5 +28,8 @@
     "*.js": [
       "eslint"
     ]
+  },
+  "scripts": {
+    "test": "cd server && yarn test && cd .. && cd client && yarn test"
```

split this into two separate commands and then merge it like 
```suggestion
    "test": "yarn test:client && yarn test:server",
    "test:client": "cd client && yarn test",
    "test:server": "cd server && yarn test"
```